mmv1/third_party/terraform/utils/utils.go

```diff
@@ -525,3 +526,28 @@ func fake404(reasonResourceType, resourceName string) *googleapi.Error {
 		Message: fmt.Sprintf("%v object %v not found", reasonResourceType, resourceName),
 	}
 }
+
+// validate name of the gcs bucket.
+func checkGCSName(name string) error {
+	MAX_LENGTH := 63
+	var err error
+	for _, str := range strings.Split(name, ".") {
+		strLen := len(str)
+		fmt.Println(str)
+		if strLen > MAX_LENGTH {
+			return fmt.Errorf("error: maximum length exceeded %v\n", str)
+		}
+		valid, _ := regexp.MatchString("^[a-z0-9]+(-[a-z0-9]+)*$", str)
```

Comment:
It seems we have had buckets with underscores be valid previously: 
`tf_test_job_extracterzjh3z33e_bucket`

Reply:
Resolved by making changes to the regex